lib/util/escapeRegExp.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * Escapes RegExp special characters from a string
```

Comment:
just cheked how eslint does it, and they just straight take regexes from users https://github.com/eslint/eslint/blob/660e0918933e6e7fede26bc675a0763a6b357c94/lib/rules/line-comment-position.js#L63

I'll commit the other changes and push, I think the current implementation is safe enough (just wildcard `*`)